### ISSUE TITLE
Request to go live. Color contrast is insufficient

### ISSUE THREAD

**Issue by amazingphilippe** [cds-snc/notification-planning#54]

**Comment by amazingphilippe** [comment]
How those links should work in inactive, hover and active states

https://www.figma.com/file/alnDnSW1Efi3Lae3uHf6Pj/Component-library-|-Bibliothèque-de-composantes?node-id=1357%3A748

![image](https://user-images.githubusercontent.com/5230720/135892750-43db53ca-b02e-444d-b0d8-47efae63a244.png)
